fusor-ember-cli/app/templates/review/progress/overview.hbs

```diff
@@ -13,3 +13,7 @@
 {{#if isCloudForms}}
   {{progress-bar model=cfmeTask name=nameCloudForms isSatelliteProgressBar=false}}
 {{/if}}
+
+{{#if showDeployTaskProgressBar}}
+  {{progress-bar model=deployTask name='Total Deployment' isSatelliteProgressBar=false}}
```

Comment:
This is the progress bar of the parent Deploy task. We only want to show it if the 3 subtasks are 100%, but the parent Deploy task is not 100%